readthedocs/core/unresolver.py

```diff
@@ -249,23 +279,23 @@ def unresolve_domain(self, domain):
             # Serve from the PUBLIC_DOMAIN, ensuring it looks like `foo.PUBLIC_DOMAIN`.
             if public_domain == root_domain:
                 project_slug = subdomain
-                log.debug("Public domain.", domain=domain)
-                return project_slug, None, False
+                log.info("Public domain.", domain=domain)
```

Comment:
```suggestion
                log.debug("Public domain.", domain=domain)
```

I think we don't want to change these logs. Probably a merge conflict.